templates/modules/mozuquotesgrid/table.hypr.live

```diff
@@ -32,24 +32,75 @@
 <div class="mz-grid-body">
     {% if model.items.length > 0 %}
         {% for row in model.gridItems %}
-            <div data-mz-row-index="{{forloop.index}}" class="mz-grid-row mz-cols-{{model.columns.length}}">
+            <div data-mz-action="selectQuoteRow" data-mz-row-index="{{forloop.index}}" class="mz-grid-row mz-cols-{{model.columns.length}}">
                 <div class="mz-grid-cols mz-cols-{{model.columns.length}}">
                   {% for col in row %}
                   <span class="mz-grid-column">
                             {{col|safe}}
                         </span>
                     {% endfor %}
                 </div>
-              {% if model.rowActions %}
+              {% if model.rowActions  %}
               <div class="mz-grid-row-actions">
                         <div class="mz-grid-column mz-quotes-grid-dropdown">
-                          <button data-mz-action="toggleDropdown" class="dropbtn"></button>
+                     
+                            {% if user.isSalesRep != true %}
+                            {% if model.items[forloop.index0].customerAccountId === model.accountId and model.isAdmin == true %}
+                                 <button data-mz-action="toggleDropdown" class="dropbtn"></button>
                             <div id="quotesDropdown{{forloop.index}}" class="dropdown-content">
                             {% for action in model.rowActions %}
-                                {% if action.isHidden != true %}
-                                    <a href="#" data-mz-action="{{action.action}}">{{action.displayName}}</a>
-                                {% endif %}
-                            {% endfor %}                                
+                                      {% if action.action == "editQuote" %}
+                                       {% if row[5] =="InReview" or row[5] == "Expired" or row[5] == "Completed" %}{% else %}
```

Comment:
These indexes might create problems in the future if we add /remove a column. think if we could do something about it.

Reply:
done